src/styles/blocks/card.scss

```
@@ -0,0 +1,78 @@
+body {
+  margin: 0;
+  font-family: $font-family;
+}
+
+.card {
+  font-family: $font-family;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+
+  &__img {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin: 16px 0 0;
+  }
+
+  &__title {
+    color: $button-big-text-color !important;
+    line-height: 18px !important;
+    font-weight: 500 !important;
+    font-size: 12px !important;
+    margin: 40px 0 0 !important;
```

```suggestion
    color: $button-big-text-color;
    line-height: 18px;
    font-weight: 500;
    font-size: 12px;
    margin: 40px 0 0;
```
Avoid using important